### PR TITLE
install: Remove existing fish_prompt by backuping

### PR DESCRIPTION
**Pull request by derekstavis**
:100: 

This avoids clean installs to start with theme issues.

### CODE REVIEW

**Comment by derekstavis** [comment]
:100:

**Comment by bobthecow** [comment]
:+1:

**Comment by derekstavis** [comment]
Rebased from master.

File: bin/install [hunk 1]
```diff
@@ -79,20 +79,31 @@ function install_omf
...
+    set -l file (basename $file_path)
+    set -l name (echo $file | cut -d. -f1)
+
     set -l timestamp (date +%s)
-    set -l original_fish_config_file "$FISH_CONFIG/config.$timestamp.copy"
+    set -l backup_file "$path/$name.$timestamp.copy"
 
-    report progress "Existent fish config file found at $fish_config_file"
-    report progress "↳ Moving file to $original_fish_config_file"
+    report progress "Conflicting $name found at $path/$file"
```
Comment (bpinto) [inline bin/install]: Conflicting -> Existent?

File: pkg/omf/cli/omf.destroy.fish [hunk 2]
```diff
@@ -5,17 +5,28 @@ function omf.destroy -d "Remove Oh My Fish"
...
 
-  set -l fish_config $XDG_CONFIG_HOME/fish
-  if test "$fish_config" = "/fish"
-    set fish_config $HOME/.config/fish
-  end
+  function omf.destroy.restore_backup -a file_path
+    set -l path (dirname $file_path)
+    set -l file (basename $file_path)
+    set -l name (echo $file | cut -d. -f1)
+    set -l candidate (echo $path/$name.*.copy | tr ' ' '\n' | sort -r | head -1)
```
Comment (bpinto) [inline pkg/omf/cli/omf.destroy.fish]: candidate -> backup_file_path?

**Comment by derekstavis** [comment]
I've done the recommended changes @bpinto

**Pull request merged**